### ISSUE TITLE
Table UI Improvement

### ISSUE THREAD

**Issue by Codipk** [ContriHUB/EAT_EASY#2]
### Expected behavior
The table should be more attractive, easy to read, and provide proper spacing and padding for better visual interpretation. Each element (text, icons, buttons) should be well-aligned and spaced out evenly to avoid clutter. Headers should be easily distinguishable from the table content, and there should be a clear indication of the status (resolved/unresolved).
Write Date in the form of DD/MM/YYYY just below the Date column and style the upvote and downvote button also show the upvotes and downvotes on a particular complaint.

### Current behavior
The table looks cramped, with little to no spacing or padding between the columns and rows. The headers and the content are not clearly distinguishable, and the alignment feels off. As a result, the table is harder to interpret and less visually appealing.Also colour the text which looks better 

### Steps to reproduce
View the "My Complaints" page with the current table structure.
Observe the close proximity of columns and lack of padding.
Notice the lack of differentiation between headers and table content.

### Possible Solution
Add padding and spacing between table rows and columns to make it more readable.
Improve header styling by adding a contrasting background color and bold text.
Increase cell padding to provide breathing space around text and icons.
Ensure proper alignment of elements (e.g., upvote/downvote icons, date).

<img width="1433" alt="Screenshot 2024-10-05 at 10 47 14 PM" src="https://github.com/user-attachments/assets/7cf595b4-8c2c-4939-aa54-1729db6c1adb">

**Comment by Vaishnavi1047** [comment]
@Codipk 
Hey, I'd like to work on this as I have some UI ideas that can help solve the problem and make it look cleaner and more organized.

**Comment by Codipk** [comment]
As per the rule who requested first on the contrihub website @AS2409, please start working on this issue. After you complete the task, create a pull request (PR) and make sure to attach a screenshot of your work on the website to showcase what you've done.Also write the comment here so that I make u assignee

**Comment by Vaishnavi1047** [comment]
@Codipk is there any new issue u have ?

**Comment by Codipk** [comment]
@Vaishnavi1047 Keep checking the website for new issue I will be keep posting.

**Comment by Codipk** [comment]
@AS2409, Hey what the status?

**Comment by AS2409** [comment]
I am getting issue in Google developer cloud credentials.

**Comment by Codipk** [comment]
@AS2409 https://dev.to/chandrapantachhetri/sending-emails-securely-using-node-js-nodemailer-smtp-gmail-and-oauth2-g3a refer this

**Comment by AS2409** [comment]
It is asking for access.

**Comment by Codipk** [comment]
Attach ss

**Comment by AS2409** [comment]
![Screenshot (190)](https://github.com/user-attachments/assets/f0c6dc28-64e5-48e7-ab0e-c23ddd8cc67f)

**Comment by Codipk** [comment]
What are you trying to do? login with google

**Comment by Codipk** [comment]
Dm me , Discord->diptikumari0337

**Comment by AS2409** [comment]
I followed the steps that were given in the documentation.

**Comment by Codipk** [comment]
@Vaishnavi1047 u can request from contrihub website

**Comment by Vaishnavi1047** [comment]
It's saying request cannot be made @Codipk

**Comment by Codipk** [comment]
@Vaishnavi1047 at a time only 3 request can be done it might possible that u have reached to 3 request.
Contact the admin of the Contrihub  from discord.

**Comment by Vaishnavi1047** [comment]
Ig I have made for the comment section ui and it wasn't accepted not
rejected

**Comment by Codipk** [comment]
@Vaishnavi1047 YES , but  that is already assign but no worries if the person not able to do that, then I'll accept yours req , But you have commented in this issue thats why i am saying

**Comment by Vaishnavi1047** [comment]
@Codipk yy sure mam as my 2 issues are in ur pending list

**Comment by kaurpreetman** [comment]
@Codipk can you please assign this issue to me.

**Comment by Codipk** [comment]
@ShouryaMishra2006, Start working on this task!

**Comment by Codipk** [comment]
Before starting go through https://github.com/ContriHUB/EAT_EASY/pull/14#issuecomment-2406958498

**Comment by ShouryaMishra2006** [comment]
@deepak-bharti-deel Thank you for assigning ! working on it

**Comment by Codipk** [comment]
At least do mentioning the correct person! @ShouryaMishra2006

**Comment by ShouryaMishra2006** [comment]
@Codipk o sorry mam thanks for assigning
